components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+import AccordionItem from '@/components/Accordion/AccordionItem';
+
+import { AccordionProps } from './Accordion.types';
+
+const Accordion: React.FC<AccordionProps> = ({ items }) => {
+  const [openKey, setOpenKey] = React.useState<string | null>(null);
```

Comment:
is there any spesific reason for you to make it only string? maybe we could keep it as a string array too so we can control it with a prop. 

even if it's not necessary for this design, we can consider we're gonna need it on another project on frontendship, maybe we will use it a UI kit. 

Choice on you!

Reply:
I actually did it so that when one opens, the other closes. Now I do this through the item. Auto power off is currently disabled.

Thanks :)